src/dotnet/APIView/APIViewWeb/Repositories/CommentsManager.cs

```diff
@@ -47,12 +88,22 @@ public async Task AddCommentAsync(ClaimsPrincipal user, CommentModel comment)
             }
         }
 
-        public async Task<CommentModel> UpdateCommentAsync(ClaimsPrincipal user, string reviewId, string commentId, string commentText)
+        public async Task<CommentModel> UpdateCommentAsync(ClaimsPrincipal user, string reviewId, string commentId, string commentText, string[] taggedUsers)
         {
             var comment = await _commentsRepository.GetCommentAsync(reviewId, commentId);
             await AssertOwnerAsync(user, comment);
             comment.EditedTimeStamp = DateTime.Now;
             comment.Comment = commentText;
+
+            HashSet<string> newTaggedUsers = new HashSet<string>();
```

Comment:
Shouldn't we populate this with current tagged users as initial value and then just add new users into this hashset to create new tagged users?

Reply:
I did it this way as the comment author might remove tagged users as well as adding them